.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-versions: [1.12.x, 1.13.x, 1.14.x]
+        platform: [ubuntu-latest, macos-latest]
```

Comment:
is windows-latest failing here too?
i'd enable it to know whether we need to log issue.

Reply:
@neolit123 i filed one already following your lead https://github.com/kubernetes/utils/issues/153

Reply:
ack, thanks